components/ProjectCard.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <div class="project-card pl-4 p-2 mr-4 mb-4">
+    <div class="flex align-items-center mb-4 mt-4">
+      <img
+        class="logo justify-start"
+        :src="project.logo"
+      >
+      <div class="ml-3">
+        <h5 class="project-heading">{{ project.name }}</h5>
+      </div>
+    </div>
+
+    <div class="description">
+      <p>{{ project.desc }}</p>
+    </div>
+
+    <div class="d-flex mt-4 justify-content-center">
+      <b-button
+        size="lg"
+        class="button mr-2 d-flex"
+        :href="project.repo"
+        target="_blank"
+      >
+        Repo
+      </b-button>
+      <b-button
+        size="lg"
+        class="button mr-2"
+        :href="project.demo"
+        target="_blank"
+      >Demo
+      </b-button>
+      <b-button
+        size="lg"
+        class="button"
+        :href="project.demo"
+        target="_blank"
+      >
+        <fa-icon
```

Comment:
I don't think we should pull fontawesome just for this play icon. Let's grab an svg